gradle/settings/all.gradle

```diff
@@ -11,10 +11,20 @@ buildscript {
 
         }
     }
+//    dependencies {
+//        classpath files("../../../gradlePlugin/build/libs/gradlePlugin-${gradlePluginsVersion}.jar")
+//        classpath "org.apache.commons:commons-lang3:${commonsLang3Version}"
+//        classpath "org.apache.commons:commons-text:1.9"
+//        classpath "commons-io:commons-io:${commonsIoVersion}"
+//        classpath "org.apache.httpcomponents:httpclient:${httpclientVersion}"
+//        classpath "org.json:json:20210307"
+//        classpath "com.fasterxml.jackson.core:jackson-databind:2.12.2"
+//        classpath "org.ajoberstar.grgit:grgit-gradle:4.1.0"
+//    }
```

Comment:
Document the purpose of this commented out block? I'm guessing similar to the settings.gradle block. Note that some of these versions are out-of-date. Should we use properties to keep them in sync as we upgrade?

Reply:
These are to run with your locally built gradle plugin (an explanation is in `settings.gradle`).
FWIW, I like to publish the gradle plugin locally (`./gradlew publishToMavenLocal`) then add `mavenLocal()` to the list of maven repositories in these files. That way the correct dependencies all get pulled in.

Reply:
Yeah, that's probably the better approach, Trey.  In the interest of speed for getting installers working again, I'm going to just update comments and add properties and look to revising the suggested approach in a later PR.